dqsegdb2/http.py

```diff
@@ -48,10 +58,29 @@ def request(url, **urlopen_kw):
     """
     if urlparse(url).scheme == 'https' and 'context' not in urlopen_kw:
         from ssl import create_default_context
-        from gwdatafind.utils import find_credential
         urlopen_kw['context'] = context = create_default_context()
-        context.load_cert_chain(*find_credential())
-    return urlopen(url, **urlopen_kw)
+        req = Request(url)
+
+        if os.environ.get('_CONDOR_CREDS'):
+          scitokens_path = os.path.join(os.environ['_CONDOR_CREDS'],'scitokens.use')
+        elif os.environ.get('SCITOKENS_FILE'):
+          scitokens_path = 'scitokens.use'
```

Comment:
This is mainly my ignorance of how scitokens is being implemented, but why is `_CONDOR_CREDS` preferred to `SCITOKENS_FILE`?

Reply:
`_CONDOR_CREDS` is an environment variable that's set by the startd to a path that contains any auth credentials that a job needs. So it might be set to `/var/condor/run/some-random-string/auth` If the user requested a scitoken, then a file is created in that directory. The name of the file is set by the condor administrator when they allow a particular issuer. In this case, I set it to `scitokens.use` as that's what we've always used. For the first iteration, I suggest leaving it at that, but if we want to get clever, the code could look through files in the `_CONDOR_CREDS` directory until it finds a valid Scitoken. That would be good code to put in the scitokens library.

The other option `SCITOKENS_FILE` is if the user just wants to point directly to a file containing a token.